public/app/plugins/datasource/cloudwatch/types.ts

```diff
@@ -501,3 +464,10 @@ export interface ResourceResponse<T> {
   accountId?: string;
   value: T;
 }
+
+export interface LogGroup {
+  arn: string;
+  name: string;
+  accountId?: string;
```

Comment:
Hey sorry my brain is still coming back after break but why does accountId and accountLabel have to be optional?

Reply:
Good questions! It's just because accountId is nullable in the [resource response](https://github.com/grafana/grafana/blob/b88b8bc291f949c6c4ea6d5f0831bfc05b08e4ba/public/app/plugins/datasource/cloudwatch/api.ts#L73). accountLabel needs to be optional as it is calculated/derived in the CrossAccountLogsQueryField component.